analyzers/src/SonarAnalyzer.CSharp/Rules/AspNet/AvoidUnderPosting.cs

```diff
@@ -144,4 +146,11 @@ type switch
 
     private static bool HasValidateNeverAttribute(ISymbol symbol) =>
         symbol.HasAttribute(KnownType.Microsoft_AspNetCore_Mvc_ModelBinding_Validation_ValidateNeverAttribute);
+
+    private static bool HasDefaultValue(SyntaxNode node) => node switch
```

Comment:
Slightly better name?
```suggestion
    private static bool IsInitialized(SyntaxNode node) => node switch
```